### DISCUSSION TITLE
I'm drawing a blank on the vocabulary of this concept

### DISCUSSION THREAD

**Discussion by jacksonrgoodman** [nss-day-cohort-47/C47-Discussions#59] [Help Me] [answered]
Does anyone know what the term for an element that looks like this is?

`${exampleObj.exampleKey}`
image example:
![Curly](https://user-images.githubusercontent.com/72589713/111489254-f6332580-8707-11eb-82b6-41093b8df1cb.PNG)

I am just trying to know what the name of the concept of using the dollar sign and curly braces, so I can better research how they work.

I am trying to see if I can use a similar concept to what we typically use in our "HTML object representations", in an alert box.

**Comment by EthanMathis** [comment, marked as answer]
I think this is a string interpolation method so you may need to put backticks around these > ${}

I could be wrong tho

**Reply by jacksonrgoodman** [reply]
Thanks! It is **string interpolation** thank you!
